_episodes/04-working-in-the-MarcEditor.md

```diff
@@ -9,10 +9,48 @@ objectives:
 - "Explain what the MarcEditor does"
 - "Explain how the MarcEditor features can help work with MARC files"
 keypoints:
-- "First key point. "
+- "MarcEditor is a tool to work with MARC data in an easily readable format"
+- "The MarcEditor can be used to perform a number of different functions such as adding, deleting MARC fields or subfields, building MARC fields, running reports, or checking the validity of MARC data"
+- "The MarcEditor can be used to perform one task at a time or automate a set of tasks for particular types of MARC files"
+- "It is necessary to break a MARC binary file to work with that MARC data in the MarcEditor. The extension of these easily readble MARC files are .mrk rather than the binary extension of .mrc"
 ---
+# Working in the MarcEditor
+MarcEditor works slightly differently than other text editors. To work with MARC data in the MarcEditor, it is first necessary to "break" the MARC binary file using MarcBreaker. To do this you can use the MarcBreaker tools. Another way is to simply click on the MARC finary file (often this file has the extension .mrc). This will prompt the MarcBreaker tool window to open. Selecting execute will start the process. Here MarcEdit reads the entire MARC file. It then creates the virtual pages of the MARC records in the file. This is called the Paging style of opening a file.
 
-{% include links.md %}
+## Paging versus Preview
+Preview mode is particularly useful for large MARC files. In Preview mode, you can use the edit functions on the file. However, the disadvantage is that you can not edit the file itself because Preview mode opens the file as read only.
 
+Paging mode is the default set when opening up a file in the MarcEditor. Here, the entire MARC file is read to retrieve the total number of records in the MARC file that are then split into a number of virtually created pages. Here, you can use any of the MarcEdtor tools and make edits to any record on any of the virtual pages. When this file is saved in Paging mode, the number of pages is recalculated, or the file is re-paged.
 
-# Working in the MarcEditor
+> ## Paging Version 7 Updates
+>
+> In version 7, the technology behind saving and re-paging saved to address the issue of the time it took to do this on large files. For previous versions, and in particular version 6, it is recommended to use the preview mode for data over approximately 150 MB. In other words, the paging mode works better for data under approximately 150 MBs.
+>
+{: .callout}
+
+# MarcEditor features
```

Comment:
Sorry I didn't get to reviewing your previous PR soon enough, but I thought that Lesson 2 was going to cover the top level features of MarcEdit (the main window), and that this lesson would go over the top level features of the MarcEditor after explaining how to break and open a file. I think it would also be useful to explain how a file is displayed and organized in  the MarcEditor since it's very different from working with records in OCLC/ILS/LSP

Reply:
I was unsure about that. In the previous lesson I covered both. That's something we can definitely take a look at.